ui/Dockerfile.dev

```diff
@@ -1,6 +1,8 @@
 # Note: This is the dockerfile for development purposes
 
-FROM node
+FROM node:14
```

Comment:
Specify (downgrade) node version to fix C compilation errors (see [here](https://github.com/nodejs/node/issues/38367)).